server/models/Admin.js

```diff
@@ -0,0 +1,26 @@
+const mongoose = require("mongoose")
+const bcrypt = require("bcryptjs")
+
+const AdminSchema = mongoose.Schema({
+  username: { type: String, required: true, index: { unique: true } },
+  password: { type: String, required: true },
+})
+
+AdminSchema.pre("save", function(next) {
+  bcrypt
+    .hash(this.password, 8)
```

Comment:
This is throwing up a linter warning, I think we should just switch the rule off. It will be setting `babel/no-invalid-this: 0`